src/components/RadioGroup/RadioGroup.spec.jsx

```diff
@@ -0,0 +1,81 @@
+import _ from 'lodash';
+import React from 'react';
+import assert from 'assert';
+// import sinon from 'sinon';
+import { mount, shallow } from 'enzyme';
+
+import getRandom from '../../util/random';
+import { common } from '../../util/generic-tests';
+
+import RadioGroup from './RadioGroup';
+
+describe('RadioGroup', () => {
+	common(RadioGroup);
+
+	describe('props', () => {
+		describe('name', () => {
+			it('sets the `name` attribute of the child radio buttons.', () => {
+				const name = getRandom();
+				const wrapper = shallow(
+					<RadioGroup name={name}>
+						<RadioGroup.RadioButton />
+						<RadioGroup.RadioButton />
+						<RadioGroup.RadioButton />
+					</RadioGroup>
+				);
+
+				_.forEach(wrapper.find('RadioButton').nodes, (node) => {
+					assert.equal(node.props.name, name);
+				});
+			});
+
+			it('defaults to a randomly generated string that is passed along to the children.', () => {
+				// Use `mount` here to get at the props that are actually passed
+				// to the component's root element (which includes the defaults)
+				// as opposed to the props that are provided by the consumer to
+				// the component.
+				const wrapper = mount(
+					<RadioGroup>
+						<RadioGroup.RadioButton />
+						<RadioGroup.RadioButton />
+						<RadioGroup.RadioButton />
+					</RadioGroup>
+				);
+				const name = wrapper.first().prop('name');
+
+				_.forEach(wrapper.find('RadioButton').nodes, (node) => {
+					assert.equal(node.props.name, name);
+				});
+			});
+		});
+
+		describe('onSelect', () => {
+			it('defaults to the Lodash `noop` method.', () => {
```

Comment:
My test suite is incomplete here. I tend to test passed in event handlers in more "functional" tests. I always test for default values in my "props" section which I think is important since the default value is a key part of the API definition, no?